loop/routes/home.js

```diff
@@ -7,9 +7,31 @@
 var loopPackageData = require('../../package.json');
 var git = require('git-rev');
 var request = require('request');
-var url = require('url')
+var url = require('url');
+var async = require('async');
+
+function isSuccess(statusCode) {
+  return statusCode === 200;
+}
 
 module.exports = function(app, conf, logError, storage, tokBox, statsdClient) {
+  function pingPushServer(pushServerURI, callback) {
+    var serverURL = url.parse(pushServerURI, false, true);
+    var protocol = serverURL.protocol === 'wss:' ? 'https' :
+      serverURL.protocol === 'ws:' ? 'http' : protocol;
```

Comment:
@kitcambridge: I think this will set `protocol` to `undefined` if `serverURL.protocol` is not "wss:" or "ws:".
Not sure if that was the intended behavior, but that last `protocol` seems confusing here since that is the current variable you're trying to set. It'd be clearer if we just did `? 'http' : undefined;` in that case.